### ISSUE TITLE
Font Awesome is no longer loading

### ISSUE THREAD

**Issue by quicoto** [ArrayHQ/atomic-blocks#29]
Hello,

WordPress: 4.9.8
Guttenberg: 4.4.0
Atomic Blocks Theme 1.2.3
Atomic Blocks - Gutenberg Blocks Collection: 1.3.8

The Font Awesome fonts are not loading for some reason. The stylesheet is not being included in the page. Here's my site https://www.ricardtorres.com/

**Isn't it because it's commented out in the code?**

Thank you

https://github.com/ArrayHQ/atomic-blocks/blob/e369c33c605af032d0f3bef79671bae91e5ddc3c/dist/init.php#L32-L43

**Comment by mikemcalister** [comment]
Hi there, 

I recently commented that out because we aren't using the fontawesome icons in the blocks yet. It was loading unnecessarily. We'll be removing that commented code until we release blocks that need icons.

**Comment by quicoto** [comment]
@mikemcalister thank you for the quick update.

I created a child theme on the Atomic Blocks Theme and had deregistred the font-awesome from there since I noticed it was being loaded twice. Back when the plugin also had them.

**Would this be something feasible from the plugin? Like, check if we already have the Atomic Theme Font Awesome to prevent double loading?**

I will remove the deregister from my child theme
